javaclient/src/main/java/confluo/rpc/MessageBatchBuilder.java

```diff
@@ -0,0 +1,61 @@
+package confluo.rpc;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+/**
+ *
+ *  A builder for a batch of message,which different from
```

Comment:
I still don't understand the need for a separate `MessageBatchBuilder`?

Reply:
a test case(testBatchWriteReadInMemory) in TestRpcClient was added,which try to batch 3 records into MessageBatchBuilder,but can't read all the record from confluod, which may cause by MessageBatchBuilder.so i try to don't care the record timestamp,batch record seperately in  MessageBatchBuilder
you may check this?